README.md

```diff
@@ -1,18 +1,21 @@
 # drone-docker
 
-Drone plugin can be used to build and publish Docker images to a container registry. For the usage information and a listing of the available options please take a look at [the docs](DOCS.md).
+[![Build Status](http://beta.drone.io/api/badges/drone-plugins/drone-docker/status.svg)](http://beta.drone.io/drone-plugins/drone-docker)
+[![Go Doc](https://godoc.org/github.com/drone-plugins/drone-docker?status.svg)](http://godoc.org/github.com/drone-plugins/drone-docker)
+[![Go Report](https://goreportcard.com/badge/github.com/drone-plugins/drone-docker)](https://goreportcard.com/report/github.com/drone-plugins/drone-docker)
+[![Join the chat at https://gitter.im/drone/drone](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/drone/drone)
+
+Drone plugin can be used to build and publish Docker images to a container
+registry. For the usage information and a listing of the available options
+please take a look at [the docs](DOCS.md).
 
 ## Build
 
 Build the binary with the following commands:
 
 ```
-export GO15VENDOREXPERIMENT=1
-export GOOS=linux
-export GOARCH=amd64
-export CGO_ENABLED=0
-
-go build -a -tags netgo
+GO15VENDOREXPERIMENT=1 CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -tags netgo
```

Comment:
we can omit GO15VENDOREXPERIMENT from the docs because Go 1.6 doesn't require the flag

also I don't think we should set the GOOS or GOARCH or -netgo tags here since someone might be testing the raw binary locally. Instead maybe this should be added to the docker build step